vulnerability-disclosure-policy.md

```diff
@@ -67,3 +67,11 @@ Reports should include:
 Please keep your vulnerability reports current by sending us any new information as it becomes available.
 
 We may share your vulnerability reports with [US-CERT](https://www.us-cert.gov/ais), as well as any affected vendors or open source projects.
+
+### Coordinated Disclosure
+
+TTS is committed to patching vulnerabilities within 90 days or less, and disclosing the details of those vulnerabilites once patches are available. We believe that public disclosure of vulnerabilities is an essential part of the vulnerability disclosure process, and that one of the best ways to make software better is to enable everyone to learn from each other's mistakes.
```

Comment:
I'm wondering about "available" here. I _think_ we actually want to wait to disclose until a fix has been _deployed_, if applicable, rather than just made available. For example, 18F/vuln-reports-private#8 has a fix, but won't be deployed for a couple of weeks, so we probably don't want to publicly disclose until after the fix hits prod. Right?

Reply:
I don't generally consider "patched privately" to be available, since that repo and issue is private, I wouldn't describe it as available.

However, if you said you'd committed a fix to a public repo, and were waiting several weeks to deploy it, I'd describe that as available, and I'd also say it's fair game to start talking about it.

Reply:
Attempted fix with https://github.com/18F/vulnerability-disclosure-policy/pull/27/commits/e37284dc33f3e0b234af6d77f0552c04df9502a4 .